node/src/proxy_client/stream_reader.rs

```diff
@@ -52,10 +51,9 @@ impl Future for StreamReader {
                     if self.logger.trace_enabled() {
                         trace!(
                             self.logger,
-                            "Read {}-byte chunk from {}: {}",
+                            "Read {}-byte chunk from {}",
                             len,
-                            self.peer_addr,
-                            utils::to_string(&Vec::from(&buf[0..len]))
```

Comment:
Can we now get rid of this function, or is it used elsewhere?

Reply:
It is still in use within the `http_packet_framer.rs` file, both in the production code and the tests.